psutil/_pssunos.py

```diff
@@ -293,7 +293,7 @@ def net_if_stats():
         isup, duplex, speed, mtu = items
         if hasattr(_common, 'NicDuplex'):
             duplex = _common.NicDuplex(duplex)
-        ret[name] = _common.snicstats(isup, duplex, speed, mtu)
+        ret[name] = _common.snicstats(isup, duplex, speed, mtu, None)
```

Comment:
let's enable this on sunos as well (see my previous comment)

Reply:
So SunOS is a completely different beast than Linux, BSD, macOS, or Windows.  Looking at the code in https://github.com/giampaolo/psutil/blob/4446e3bd5fd29e0598f2ca6672d5ffe8ca40d442/psutil/_psutil_sunos.c#L1484 , it looks like it doesn't even use the same `ioctl` (it is using something called SIOCGLIFFLAGS), and then it verifies that data with another structure (kstat?).  So I don't think I can easily enable SunOS, and I don't have access to a SunOS machine.